ospo-mindmap/Content/ospomindmap.md

```diff
@@ -307,8 +306,9 @@ incubation and post launch
 - Increase productivity, quality, code re-use, transparency and trust
 - Establish communication channels across the organization
 - Improve collaboration across the organization
+  - Help navigate internal politics or policies
```

Comment:
I do see how "Help navigate internal politics or policies" could be seen as sub point of "Improve collaboration across the organization" / InnerSource section. If we adding this as a subpoint I would prefer if we add another example else it look weird to me in the rendered mind map